assets/js/modules/analytics/util/countries-data.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sitekit/camelcase-acronyms */
```

Comment:
Regarding my previous comments, this file is one of the special cases where a file-wide exception is acceptable.